docs/GettingStarted.md

```diff
@@ -340,12 +341,13 @@ For a list of available integrations, and their configuration options, please re
 | Grape                    | `grape`                    | `>= 1.0`                 | *[Link](#grape)*                    | *[Link](https://github.com/ruby-grape/grape)*                                  |
 | GraphQL                  | `graphql`                  | `>= 1.7.9`               | *[Link](#graphql)*                  | *[Link](https://github.com/rmosolgo/graphql-ruby)*                             |
 | gRPC                     | `grpc`                     | `>= 1.10`                | *[Link](#grpc)*                     | *[Link](https://github.com/grpc/grpc/tree/master/src/rubyc)*                   |
+| HTTParty                 | `httparty`                 | `>= 0.17`                | *[Link](#httparty)*                 | *[Link](https://github.com/jnunemaker/httparty)*                               |
```

Comment:
It's very likely not all our users are up-to-date with their version of `httparty`.

Could you please verify what's the minimum version of `httparty` we can support with the implementation this PR?

Just to be clear, I don't expect you to make changes to accommodate all old versions of `httparty`, but it's very likely that we support older version without changing the implementation, we just have to double check that the code on older versions of `httparty` is compatible.

Reply:
I would say this was one area of general confusion that our contribution guidelines could offer more guidance on.

Reply:
Looks like we can go as far back as `0.14.0` with this implementation.